### DISCUSSION TITLE
Nextcloud integration

### DISCUSSION THREAD

**Discussion by terraboss** [Stirling-Tools/Stirling-PDF#413] [Ideas]
Hello, 

Is there a Nextcloud app for Stirling PDF that I don't know about yet? Or is there another way to integrate Stirling PDF into Nextcloud for editing PDFs? 

I work a lot and like to work with PDFs. 
An integration would be great.

Many greetings

**Comment by Frooodle** [comment]
Sorry there is no direct app or support for nextcloud 

Stirling pdf does have a full API which can be used and integrated with applications but no dedicated apps etc

**Reply by terraboss** [reply]
Thanks for the fast reply. 

Than, unfortunately, I guess Stirling PDF isn't the right option for me.

**Comment by MickGe** [comment]
Maybe it could be done with https://apps.nextcloud.com/apps/external.
